src/ccache.cpp

```diff
@@ -1106,6 +1103,7 @@ get_result_key_from_cpp(Context& ctx, Args& args, Hash& hash)
   if (!ctx.config.run_second_cpp()) {
     // If we are using the CPP trick, we need to remember this stderr data and
     // output it just before the main stderr from the compiler pass.
+    ctx.cpp_stdout = stdout_path;
```

Comment:
At this point the `stdout_path` contains the output of the preprocessor and must not be printed to the console.  

The variable `stdout_path` should perhaps also be renamed so that no further confusion is caused here. (It can also be the path to a *.i file.)

With MSVC it is not clear to me what happens with the flag /E. Are then also still error texts in the stdout which must be printed here?

Reply:
I just don't understand why the msvc build here doesn't also write the preprocessor output to the console. @llunak can you explain or imagine why this is?

Reply:
I've just submitted #970 which makes this moot.